bayesian_testing/experiments/delta_normal.py

```diff
@@ -0,0 +1,298 @@
+
+    @property
+    def sum_values(self):
+        return [self.data[k]["sum_values"] for k in self.data]
+
+    @property
+    def sum_values_2(self):
+        return [self.data[k]["sum_values_2"] for k in self.data]
+
+    @property
+    def a_priors_beta(self):
+        return [self.data[k]["a_prior_beta"] for k in self.data]
+
+    @property
+    def b_priors_beta(self):
+        return [self.data[k]["b_prior_beta"] for k in self.data]
+
+    @property
+    def m_priors(self):
+        return [self.data[k]["m_prior"] for k in self.data]
+
+    @property
+    def a_priors_ig(self):
+        return [self.data[k]["a_prior_ig"] for k in self.data]
+
+    @property
+    def b_priors_ig(self):
+        return [self.data[k]["b_prior_ig"] for k in self.data]
+
+    @property
+    def w_priors(self):
+        return [self.data[k]["w_prior"] for k in self.data]
+
+    def eval_simulation(
+            self, sim_count: int = 20000, seed: int = None, min_is_best: bool = False
+    ) -> Tuple[dict, dict]:
+        """
+        Calculate probabilities of being best and expected loss for a current class state.
+
+        Parameters
+        ----------
+        sim_count : Number of simulations to be used for probability estimation.
+        seed : Random seed.
+        min_is_best : Option to change "being best" to a minimum. Default is maximum.
+
+        Returns
+        -------
+        res_pbbs : Dictionary with probabilities of being best for all variants in experiment.
+        res_loss : Dictionary with expected loss for all variants in experiment.
+        """
+        pbbs, loss = eval_delta_normal_agg(
+            self.totals,
+            self.positives,
+            self.sum_values,
+            self.sum_values_2,
+            sim_count=sim_count,
+            a_priors_beta=self.a_priors_beta,
+            b_priors_beta=self.b_priors_beta,
+            m_priors=self.m_priors,
+            a_priors_ig=self.a_priors_ig,
+            b_priors_ig=self.b_priors_ig,
+            w_priors=self.w_priors,
+            seed=seed,
+            min_is_best=min_is_best,
+        )
+        res_pbbs = dict(zip(self.variant_names, pbbs))
+        res_loss = dict(zip(self.variant_names, loss))
+
+        return res_pbbs, res_loss
+
+    def evaluate(
+        self, sim_count: int = 20000, seed: int = None, min_is_best: bool = False
+    ) -> List[dict]:
+        """
+        Evaluation of experiment.
+
+        Parameters
+        ----------
+        sim_count : Number of simulations to be used for probability estimation.
+        seed : Random seed.
+        min_is_best : Option to change "being best" to a minimum. Default is maximum.
+
+        Returns
+        -------
+        res : List of dictionaries with results per variant.
+        """
+        keys = [
+            "variant",
+            "totals",
+            "positives",
+            "sum_values",
+            "avg_values",
+            "avg_positive_values",
```

Comment:
positive values here are likely not correct for similar reasons as above. So maybe `avg_non_zero_values`